environments/mongo.js

```diff
@@ -0,0 +1,6 @@
+module.exports = {
```

Comment:
Frameworks is admittedly a bad name, but this belongs there, and mixed into some other config that has the standard backend configuration.